test/test_productid.py

```diff
@@ -9,7 +9,6 @@
 import os
 import shutil
 import tempfile
-import yum
```

Comment:
Is removing of yum in this test related to removing Alex's effort of removing dependency to yum?

Reply:
No, this is just dead code removal, resulting in a no-longer-needed import 😄